addons/hot/CfgWeapons.hpp

```diff
@@ -0,0 +1,59 @@
+class CfgWeapons {
+    class RocketPods;
+    class MissileLauncher;
+    class GVAR(1_launcher): RocketPods {
+        displayName = "HOT 1";
+        magazines[] = {QGVAR(1_6Rnd), QGVAR(1_2Rnd), QGVAR(1_PylonMissile_1Rnd), QGVAR(1_PylonRack_1Rnd), QGVAR(1_PylonRack_3Rnd), QGVAR(1_PylonRack_4Rnd)};
+        initSpeed = 100;
+        autoFire = 0;
+        canLock = 0;
+        weaponLockSystem = 0;
+        lockingTargetSound[] = {"",0,1};
+        lockedTargetSound[] = {"",0,1};
+        soundFly[] = {"A3\Sounds_F\weapons\Rockets\rocket_fly_1",1,1.1,700};
+        nameSound = "MissileLauncher";
+        sounds[] = {"StandardSound"};        
+        class StandardSound {
+            begin1[] = {"A3\Sounds_F\weapons\Rockets\missile_1",1.12202,1.3,1000};
+            soundBegin[] = {"begin1",1};
+            soundsetshot[] = {"RocketsMedium_Shot_SoundSet"};
+        };
+        cursor = "EmptyCursor";
+        cursorAim = "missile";
+        showAimCursorInternal = 0;
+    };
+    class GVAR(2_launcher): GVAR(1_launcher) {
+        displayName = "HOT 2";
+        magazines[] = {QGVAR(2_6Rnd), QGVAR(2_2Rnd), QGVAR(2_PylonMissile_1Rnd), QGVAR(2_PylonRack_1Rnd), QGVAR(2_PylonRack_3Rnd), QGVAR(2_PylonRack_4Rnd)};
+    };
+    class GVAR(2mp_launcher): GVAR(1_launcher) {
+        displayName = "HOT 2MP";
+        magazines[] = {QGVAR(2MP_6Rnd), QGVAR(2MP_2Rnd), QGVAR(2MP_PylonMissile_1Rnd), QGVAR(2MP_PylonRack_1Rnd), QGVAR(2MP_PylonRack_3Rnd), QGVAR(2MP_PylonRack_4Rnd)};
+    };
+    class GVAR(3_launcher): GVAR(1_launcher) {
+        displayName = "HOT 3";
```

Comment:
needs translation

Reply:
Do all these really need translations?
I don't think it's going to be `HEISS 3` in german :smile:

Reply:
Because it is the name of the missile does it need translation? You wouldn't translate AGM-114 or MILAN because that is the missile name.

Reply:
I'm thinking about possible chinese and japanese translations

Reply:
You mean `HEIẞ 3`